### DISCUSSION TITLE
Hola !

### DISCUSSION THREAD

**Discussion by SpartanEsp** [Ludeon/RimWorld-Spanish#86] [General]
Hola, veo que hace meses que no se hace ni una sola corrección ni actualización, se ha abandonado la traducción ?

Dispongo de algo de tiempo para ir aportando cosillas, puedo colaborar ?

**Comment by Zakees** [comment]
Buenas. Ya vi que estuviste progresando y tal, a modo de reflexión, muchas veces estamos ocupados en otras cosas y desatendemos completamente esto durante un tiempo. Evidentemente la traducción ha de seguir avanzando por lo que toda ayuda es más que bienvenida. 
Por lo general, tanto yo como Zerstrick estamos disponibles en Discord para echar una mano, porque el repositorio es más difícil de comprobar periódicamente, por lo que cualquiera puede acudir a nosotros para cualquier cuestión de la traducción y poder echar una manita ^^

Esto lo cierro ya, gracias por participar en la traducción :)
